DESCRIPTION

```diff
@@ -53,9 +52,12 @@ Suggests:
     DBI (>= 0.7),
     RSQLite (>= 2.0),
     RPostgres (>= 1.4.4),
+    arrow (>= 12.0.1),
     webshot (>= 0.5.0), 
     testthat (>= 2.0.0),
     pkgdown (>= 1.1.0)
+Additional_repositories:
+    https://p3m.dev/cran/2024-02-02
```

Comment:
CRAN acknowledges this field but doesn't use it. `install.packages()` does not use it. `remotes::install_*()` does use it, but `pak::pak()` does not (yet).

Alternatively, you can instead get the latest pre-release version of arrow by instead using `Additional_repositories: https://apache.r-universe.dev/` or `Additional_repositories: https://nightlies.apache.org/arrow/r`.